.studiorc

```diff
@@ -41,12 +41,12 @@ export ELASTICSEARCH_URL="http://localhost:10141"
 export POSTGRESQL_URL="localhost:5432"
 # Ingest
 export INGEST_URL="https://localhost:10122"
-export INGEST_PORT=$(echo $INGEST_URL | awk -F: '{ print $NF }')
-export INGEST_HOST=$(echo $INGEST_URL | awk -F: '{ print $2 }' | tr -d '/')
+export INGEST_PORT="localhost"
```

Comment:
I've found this a bit silly, so I've ended it 😄

Reply:
Uh oh, are these reversed?  Should this be 10122 and the next one "localhost"?

Reply:
😅 oh no!

I'll fix that.

Reply:
https://github.com/chef/automate/pull/933